app/pages/sections/additional-services/select/routes.test.js

```diff
@@ -321,4 +322,37 @@ describe('additional-services select routes', () => {
       expect(res.text.includes('data-test-id="error-title"')).toBeFalsy();
     });
   });
+
+  describe('GET /organisation/:orderId/additional-services/select/additional-service/price/recipient', () => {
+    const path = '/organisation/some-order-id/additional-services/select/additional-service/price/recipient';
+
+    it('should redirect to the login page if the user is not logged in', () => (
+      testAuthorisedGetPathForUnauthenticatedUser({
+        app: request(setUpFakeApp()), getPath: path, expectedRedirectPath: 'http://identity-server/login',
+      })
+    ));
+
+    it('should show the error page indicating the user is not authorised if the user is logged in but not authorised', () => (
+      testAuthorisedGetPathForUnauthorisedUser({
+        app: request(setUpFakeApp()),
+        getPath: path,
+        getPathCookies: [mockUnauthorisedCookie],
+        expectedPageId: 'data-test-id="error-title"',
+        expectedPageMessage: 'You are not authorised to view this page',
+      })
+    ));
+
+    it('should return the additional-services select recipient page if authorised', async () => {
+      getSolutionHelper.getSolution = jest.fn()
+        .mockResolvedValue({});
+
+      const res = await request(setUpFakeApp())
+        .get(path)
+        .set('Cookie', [mockAuthorisedCookie])
+        .expect(200);
+
+      expect(res.text.includes('data-test-id="additional-service-recipient-page"')).toBeTruthy();
+      expect(res.text.includes('data-test-id="error-title"')).toBeFalsy();
+    });
+  });
```

Comment:
👍  didn't spot this route was in the wrong routes file earlier

Reply:
Oh dear was this my doing?